server/src/main/java/org/apache/druid/server/coordinator/duty/KillUnusedSegments.java

```diff
@@ -300,7 +311,10 @@ private boolean canDutyRun()
   private int getAvailableKillTaskSlots(final CoordinatorDynamicConfig config, final CoordinatorRunStats stats)
   {
     final int killTaskCapacity = Math.min(
-        (int) (CoordinatorDutyUtils.getTotalWorkerCapacity(overlordClient) * Math.min(config.getKillTaskSlotRatio(), 1.0)),
+        (int) (CoordinatorDutyUtils.getTotalWorkerCapacity(overlordClient) * Math.min(
+            config.getKillTaskSlotRatio(),
+            1.0
+        )),
```

Comment:
Nit: for better readability
```suggestion
        (int) (CoordinatorDutyUtils.getTotalWorkerCapacity(overlordClient)
               * Math.min(config.getKillTaskSlotRatio(), 1.0)),
```

Reply:
Oops, changed. This must have got caught when I applied formatting since it was over 120 chars.